test/restart/test_uda.py

```diff
@@ -0,0 +1,110 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""
+Test UDAs across restart.
+"""
+import sys
+import pytest
+
+
+# Import fixtures from topology
+sys.path.insert(1, sys.path[0] + '/..')
+from topology.conftest import manager
+
+# Use the util.py library from ../cql-pytest:
+sys.path.insert(1, sys.path[0] + '/../cql-pytest')
+from util import unique_name
+from cassandra.protocol import InvalidRequest
+
+@pytest.mark.asyncio
+async def test_uda_restart(manager):
+    """Test UDA survives server restart"""
+    servers = await manager.servers()
+    schema = 'pk int PRIMARY KEY, tx text'
+    keyspace = unique_name()
+    manager.cql.execute(f"CREATE KEYSPACE {keyspace} WITH REPLICATION = {{ 'class' : 'SimpleStrategy', 'replication_factor' : 1 }}")
+
+    # Prepare a test table
+    table = keyspace + "." + unique_name()
+    manager.cql.execute(f"CREATE TABLE {table} ({schema})")
+    for x in range(1, 100):
+        s = "a" * x
+        manager.cql.execute(f"INSERT INTO {table} (pk, tx) VALUES ({x}, '{'a' * x}');")
```

Comment:
Nitpick: I'm not sure this test needs any content at all in the table (wouldn't the uda fail even without content), but if it does, I think 100 is not really necessary and I think doesn't improve the test over 10 or whatever.

Reply:
I decreased this number to 10 in the dtest